docs/sources/getting-started/build-influxdb-dashboard.md

```diff
@@ -58,7 +57,7 @@ InfluxDB publishes guidance for connecting different versions of their product t
 - If the text name of the organization or bucket doesn't work, then try the ID number.
 - If you change your bucket name in InfluxDB, then you must also change it in Grafana and your Telegraf .conf file as well.
 
-## Step 6. Add a query
+## Add a query
```

Comment:
would switch these to h3 for readability

Reply:
Made them H4s, see how you like it.